### PR TITLE
[no-std] Add Documentation in README

### PR DESCRIPTION
**Pull request by DemesneGH**

### CODE REVIEW

**Comment by jbech-linaro** [comment]
`Acked-by: Joakim Bech <joakim.bech@linaro.org>`

File: README.md [hunk 1]
```diff
@@ -25,11 +33,47 @@ Teaclave (incubating)](https://teaclave.apache.org/).
...
 
+
+## TA Development Modes
+
+### Comparison
+
+#### `no-std`
+
+- **Pros**:
+  - Reuses standard Rust AArch64 tier-1 toolchain target (`aarch64-unknown-linux-gnu`).
```
Comment (b49020) [inline README.md:50]: This is true for 32-bit systems (AArch32) as well where toolchain target is `arm-unknown-linux-gnueabihf`.

**Review by b49020 — COMMENTED** [review]
Apologies I never got around to improving documentation due to other commitments. Apart from a minor comment changes looks good to me.

With that comment incorporated, feel free to add:

`Reviewed-by: Sumit Garg <sumit.garg@linaro.org>`

Also, I suppose similar change should be applied to `master` branch as well.

**Review by Sword-Destiny — COMMENTED** [review]
/lgtm

**Comment by DemesneGH** [comment]
@Sword-Destiny Could you give me a `Reviewed-by` tag like above? Thanks!

**Comment by DemesneGH** [comment]
> I suppose similar change should be applied to `master` branch as well.

@b49020 Thanks for your comment. Sure, I will sync to `master` after this PR merged.

Will also start the process for setting the `default` branch on the community.

File: README.md (hunk 1 above)
Comment (DemesneGH) [reply README.md]: done

**Comment by DemesneGH** [comment]
synced to `master`: https://github.com/apache/incubator-teaclave-trustzone-sdk/commit/cd19ac2e1c3cb1a848d5131d4af8138d84be8708

**Comment by DemesneGH** [comment]
@b49020 Could you help to review the new PR https://github.com/apache/incubator-teaclave-trustzone-sdk/pull/131 (with some doc and ci changes)? thanks!
BTW seems I cannot mention(@) you in the PR you've not participated in, so send the comment here :)

**Pull request merged**